README.md

```diff
@@ -0,0 +1,103 @@
+![](https://img.shields.io/badge/microverse-blueviolet)
+
+# Vacation Booking App(Front-end)
+
+> Vacation Booking App is a tool that helps users to plan for thier vacation trips. User can view a list of beautiful destinations, register and choose one or many they want to book a trip for. Regestered users are able to view a list of destinations they booked. Only admin users are allowed to add destinations.
+
+![vacations-booking-screenshot](https://user-images.githubusercontent.com/14215157/143226630-4772855a-9640-4aaf-a595-11be928e8e5a.png)
```

Comment:
You can do a better job of describing the project, maybe add better screenshots and different pages featured.